rpcs3/Emu/Cell/lv2/sys_ss.cpp

```diff
@@ -88,7 +89,27 @@ error_code sys_ss_random_number_generator(u64 pkg_id, vm::ptr<void> buf, u64 siz
 	if (!rnd || rnd.read(temp.get(), size) != size)
 #endif
 	{
-		fmt::throw_exception("sys_ss_random_number_generator(): Failed to generate pseudo-random numbers" HERE);
+		sys_ss.error("sys_ss_random_number_generator(): Failed to generate pseudo-random numbers");
```

Comment:
Need more info, /dev/urandom isn't supposed to fail, for example. Maybe repeating CryptGenRandom.